lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -798,38 +778,42 @@ export class RunsOverviewModel extends Observable {
     }
 
     /**
-     * Retro-compatibility access to paginated runs, returning all the runs contained in the current page if it applies
+     * Return all the runs currently filtered, without paging
      *
-     * @return {RemoteData} the runs in the current page
+     * @return {RemoteData} the remote data of the runs
      */
-    get runs() {
-        return this._currentPageRuns;
+    get exportableItems() {
+        return this._exportableItems.getCurrent();
     }
 
     /**
-     * Return all the runs currently filtered, without paging
+     * Update the cache containing all the runs without paging
      *
-     * @return {RemoteData} the remote data of the runs
+     * @return {Promise<void>} void
+     * @private
      */
-    get allRuns() {
-        if (this._allRuns.isNotAsked()) {
-            this._fetchAllRunsWithoutPagination();
+    async loadExportableRuns() {
```

Comment:
You are completely breaking the previous logic here, are you aware of it?
Before, we fetch all runs only if the current list of runs can't be reused as export.

Reply:
and it is so now, you can't reuse iff you changed filtering and this iff `root endpoint` is different - see condition if `loadExportRuns` (previously: `loadExportableRuns`)